src/main/kotlin/no/nav/medlemskap/clients/aareg/AaRegClient.kt

```diff
@@ -57,13 +58,14 @@ class AaRegClient(
             }
         )
     }
-/*
+
     suspend fun healthCheck(): HttpResponse {
         return httpClient.get {
-            url("$baseUrl/aareg-services/api/ping")
+            url("$baseUrl/ping")
+            header(HttpHeaders.Accept, ContentType.Text.Plain)
             header("Nav-Consumer-Id", username)
```

Comment:
Ikke behov for autentisering i header?

Reply:
Står ikke noe om autentisering i dokumentasjonen, er ikke helt sikker men AaReg inneholder vel bare data som allerede er offentlig, så da er det kanskje ikke så farlig?